manifests/certonly.pp

```diff
@@ -45,6 +45,7 @@
   Letsencrypt::Plugin                       $plugin               = 'standalone',
   Array[Stdlib::Unixpath]                   $webroot_paths        = [],
   String[1]                                 $letsencrypt_command  = $letsencrypt::command,
+  Optional[Integer]                         $key_size             = $letsencrypt::params::key_size,
```

Comment:
Since you have a default this can simply be `Integer`. Probably good to use `Integer[2048]` since we really don't want a smaller key size. It's also better to use `$letsencrypt::key_size` so users can override the default.

Reply:
Done as requested.